### PR TITLE
NFDIV-4197: Refactor code, use new Welsh templates

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Refactor code, use new Welsh templates

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-4197

### Pull request checklist ###

**Before raising**
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**Before merging**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

**Note:** Bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -55,7 +55,7 @@ uk:
         templates:
           english:
             SAVE_SIGN_OUT: '3aff9dc0-0188-42f6-a0be-fc5daa404d9a'
-            OUTSTANDING_ACTIONS: '544c4297-e1c5-44bc-8f02-5661494beb4f'
+            OUTSTANDING_ACTIONS: 'b077b50a-6fe2-48bd-aac5-2da5deceac86'
```
Comment (adamg-hmcts) [inline src/main/resources/application.yaml:58]: Is this the right English template as it looks like it's formatted differently to the Welsh and has slightly different content?

I think the brackets are unmatched part way down the template at `Apply to serve the ((divorceOrDissolution??divorce)) papers another way))`

File: src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java [hunk 2]
```diff
@@ -99,25 +99,27 @@ private Map<String, String> applicant1TemplateVars(final CaseData caseData, fina
...
+        templateVars.putAll(courtDocumentDetails(caseData));
         templateVars.putAll(serveAnotherWayTemplateVars(false, caseData));
         return templateVars;
     }
 
     private Map<String, String> serveAnotherWayTemplateVars(boolean soleServingAnotherWay, CaseData caseData) {
         Map<String, String> templateVars = new HashMap<>();
 
+        boolean servedAnotherWay = soleServingAnotherWay && caseData.isDivorce();
+        String divorceOrDissolution = caseData.isDivorce() ? "divorce" : "";
```
Comment (adamg-hmcts) [inline src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java:111]: Does this get adjusted for the Welsh versions?

File: src/main/resources/application.yaml (hunk 1 above)
Comment (FaisalMoJ) [reply src/main/resources/application.yaml:58]: I have fixed issues with Welsh template after translation provided

File: src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java (hunk 2 above)
Comment (FaisalMoJ) [reply src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java]: Added code to get Welsh text for Welsh language preference

**Pull request merged**